src/ralph/api/routers/statements.py

```diff
@@ -504,7 +502,9 @@ async def post(
         _enrich_statement_with_authority(statement, current_user)
 
     try:
-        existing_statements = DATABASE_CLIENT.query_statements_by_ids(statements_ids)
+        existing_statements = DATABASE_CLIENT.query_statements_by_ids(
+            list(statements_dict.keys())
```

Comment:
Even if it's less explicit, the following should do the trick:

```suggestion
            list(statements_dict)
```